client/src/modules/patients/registry/editPatientGroup.js

```diff
@@ -0,0 +1,50 @@
+angular.module('bhima.controllers')
+  .controller('EditPatientGroupModalController', EditPatientGroupModalController);
+
+EditPatientGroupModalController.$inject = [
+  'data', '$uibModalInstance', 'NotifyService',
+  'PatientGroupService', 'PatientService',
+];
+
+/**
+ * @class EditPatientGroupModalController
+ */
+function EditPatientGroupModalController(data, Instance, Notify, PatientGroup, Patient) {
+  const vm = this;
+  vm.cancel = () => Instance.close();
+  vm.submit = submit;
+  vm.subscribedGroups = {};
+
+  PatientGroup.read().then(sessionGroups => {
+    vm.patientGroups = sessionGroups;
+    sessionGroups.forEach((patientGroup) => {
+      vm.subscribedGroups[patientGroup.uuid] = false;
+    });
+  }).catch(Notify.handleError);
+
+  function getGroups() {
+    const subscribedGroups = [];
+    const keys = Object.keys(vm.subscribedGroups);
+    keys.forEach(groupUuid => {
```

Comment:
Just FYI - angular has a shortcut for this.  Instead of doing:
```js
const keys = Object.keys(someObject);
keys.forEach(key => { 
  console.log(someObject[key]);
});
```
you can do the same thing in a single line:
```js
angular.forEach(someObject, (value, key) => {
  console.log(key, value);
});
```

I personally don't really care which is used, but figured I would mention it in case you didn't know!